Classes/Settings/SettingsViewController.swift

```diff
@@ -84,15 +84,18 @@ GitHubSessionListener {
             case .success(let response):
                 let text: String
                 let color: UIColor
-                switch response.data.status {
-                case .good:
-                    text = NSLocalizedString("Good", comment: "")
+                switch response.data.status.indicator {
+                case .none:
+                    text = response.data.status.description
                     color = Styles.Colors.Green.medium.color
```

Comment:
You could add a computed property on `StatusType` called color and this `switch` could be removed.

Reply:
I decided not to do this as this would need `StatusType` to import `UIKit`, which feels unnecessary, and a weird dependency in the networking layer.